packages/@expo/cli/e2e/__tests__/install-test.ts

```diff
@@ -147,7 +147,7 @@ it(
   'runs `npx expo install --fix` fails',
   async () => {
     const projectRoot = await setupTestProjectAsync('install-fix-fail', 'with-blank');
-    await installAsync(projectRoot, ['add', 'expo-sms@1.0.0', 'expo-auth-session@1.0.0']);
```

Comment:
I saw that `installAsync` actually runs `bun install ...`, meaning that we were installing the `add` dependencies here 🙈